cmd/echoServer.cc

```diff
@@ -3,19 +3,23 @@
 #include <sstream>
 #include <thread>
 
-#include "cmdLogger.h"
 #include <transport/transport.h>
+#include "cantina/logger.h"
```

Comment:
Shouldn't this be using <> instead of ""?

Reply:
No. The others should be "transport/transport.h".  The <> is for system include files, whereas "" is for non-system include files.

Reply:
Changed.